doc-content/shared-kie-docs/src/main/asciidoc/Workbench/SecurityManagement/business-central-security-management-provider-con.adoc

```diff
@@ -0,0 +1,51 @@
+[id='business-central-security-management-provider-con']
+
+= Security management providers
+
+A security environment is provided by the use of a realm. Realms are used to restrict access to the different application resources. These realms contains the information about the users, groups, roles, and permissions. A security provider is the formal name given to a concrete user and group management service implementation for a given realm.
+
+The user & group management features available will depend on the security provider configured. If the built-in providers do not fit with your application security realm, then you can easily build and register your own security management provider.
+
+NOTE: If the security management provider is not installed, the user interface for managing the security realm will not be available. Once a management security provider is installed and setup, the user and group management features are automatically enabled in the security management user interface.
+
+{CENTRAL} provides the following security management providers:
+
+* Wildfly or {EAP} security management provider
++
+This security provider supports realm types based on `application-users.properties` or `application-roles.properties` property files.
++
+You can use the following system properties for the Wildfly or {EAP} security management provider implementation:
++
+.Wildfly or {EAP} security management provider properties
+[%header,cols=2]
+[%autowidth]
+|===
+|Property |Description
+
+|`*org.uberfire.ext.security.management.wildfly.properties.WildflyUserManagementService*` | You can use this property for user, group, and role implementations.
+
+|`*org.uberfire.ext.security.management.wildfly.properties.WildflyUserPropertiesManager*` | You can use this property for user management implementations. This property maps the Wildfly or {EAP} users for the given realm to the application users.
+
+|`*org.uberfire.ext.security.management.wildfly.properties.WildflyGroupPropertiesManager*` | You can use this property for group management implementations. This property maps the Wildfly or {EAP} roles for the given realm to the application groups.
+
+|===
+
+* Tomcat security management provider
++
+This provider supports realm types based on `XML` files. The default value is tomcat server, such as `tomcat-users.xml`. The classes used in this package contain the user and group manager implementations for Apache Tomcat.
```

Comment:
@kaldesai Instead of "The default value is tomcat server, such as `tomcat-users.xml`", we can replace it with something like "The default file in tomcat server is tomcat-users.xml".